gp-templates/translations.php

```diff
@@ -26,6 +26,17 @@
 	<?php endif; ?>
 </h2>
 <?php
+
+/**
+ * Fires header before toolbar
+ *
+ * @since 3.0.0
+ *
+ * @param GP_Translation_Set $translation_set The current translation set.
+ * @param GP_Project $project The current project.
+ */
+do_action( 'gp_translation_view_message', $translation_set, $project );
```

Comment:
This seems unrelated and should be part of a separate PR.

Reply:
You're right.